pages/common/apg.md

```diff
@@ -0,0 +1,23 @@
+# apg
+
+> Creates arbitrarily complex random passwords.
+
+- Create random passwords (default password length is 8):
+
+`apg`
+
+- Create a password with at least 1 symbol (S), 1 number (N), 1 uppercase (C), 1 lowercase (L):
+
+`apg -M SNCL`
+
+- Create a password with 16 characters:
+
+`apg -m {{16}}`
+
+- Create a password with maximum length of 16:
+
+`apg -x {{16}}`
+
+- Create a password that doesn't appear in a dictionary (you have to provide the dictionary):
```

Comment:
side note: this should be added to CONTRIBUTING.md. [Wikipedia has some wording](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Second-person_pronouns) we might take inspiration from.